groups/netapp-insight/data.tf

```diff
@@ -0,0 +1,53 @@
+data "aws_vpc" "vpc" {
+  tags = {
+    Name = "vpc-${var.aws_account}"
+  }
+}
+
+data "aws_subnet_ids" "monitor" {
+  vpc_id = data.aws_vpc.vpc.id
+  filter {
+    name   = "tag:Name"
+    values = ["sub-monitor-*"]
+  }
+}
+
+data "aws_route53_zone" "private_zone" {
+  name         = local.internal_fqdn
+  private_zone = true
+}
+
+data "aws_kms_key" "ebs" {
+  key_id = "alias/${var.account}/${var.region}/ebs"
+}
+
+data "vault_generic_secret" "account_ids" {
+  path = "aws-accounts/account-ids"
+}
+
+data "vault_generic_secret" "internal_cidrs" {
+  path = "aws-accounts/network/internal_cidr_ranges"
+}
+
+data "vault_generic_secret" "netapp-insight" {
+  path = "applications/${var.aws_account}-${var.aws_region}/netapp/${var.application}"
+}
+
```

Comment:
Needs the following for SSM access and logging:

data "vault_generic_secret" "security_kms_keys" {
  path = "aws-accounts/security/kms"
}

data "vault_generic_secret" "security_s3_buckets" {
  path = "aws-accounts/security/s3"
}